src/calculateNodeHeight.ts

```diff
@@ -61,5 +66,5 @@ export default function calculateNodeHeight(
   }
   height = Math.min(maxHeight, height);
 
-  return height;
+  return { height, rowHeight };
```

Comment:
could we make this a tuple? the final code would be slightly smaller 😉 TS 4.0 will come for labels for tuples so we'll be able to improve readability for this then

Reply:
Sure, I wasn't sure if you wanted to expose more values.